apis/serving/v1alpha1/servingruntime_types.go

```diff
@@ -183,3 +188,7 @@ func init() {
 func (sr ServingRuntime) Disabled() bool {
 	return sr.Spec.Disabled != nil && *sr.Spec.Disabled
 }
+
+func (sr ServingRuntime) IsModelMeshCompatible() bool {
```

Comment:
Maybe rename this to `IsMultiModelRuntime`?

Reply:
Yea, I think that name fits better.